packages/console/src/components/FeatureTag/index.tsx

```diff
@@ -72,18 +83,22 @@ export type Props = {
  * ```
  */
 function FeatureTag(props: Props) {
-  const { className } = props;
+  const { className, isEnterprise } = props;
   const { isDevTenant } = useContext(TenantsContext);
 
-  const { isVisible, plan } = props;
+  const { isVisible } = props;
 
   // Dev tenant should always see the tag since they have access to almost all features, and it's
   // useful for developers to know which features need to be paid for in production.
   if (!isDevTenant && !isVisible) {
     return null;
   }
 
-  return <div className={classNames(styles.tag, className)}>{planIdTagMap[plan]}</div>;
+  if (isEnterprise) {
+    return <div className={classNames(styles.tag, className)}>{planTagMap.enterprise}</div>;
+  }
+
+  return <div className={classNames(styles.tag, className)}>{planTagMap[props.plan]}</div>;
```

Comment:
```
return <div className={classNames(styles.tag, className)}>{isEnterprise 
 ? planTagMap.enterprise : planTagMap[props.plan]}</div>;
```

Reply:
Can not properly infer the type, if write in your way

Reply:
If `isEnterprise` is specified, no need to input `plan`; but `plan` is required if no `isEnterprise` is input.

Reply:
ok